javascript-data-model/order-history.js

```diff
@@ -0,0 +1,91 @@
+var orderHistory = [
+  {
+    item: 'JavaScript for impatient programmers',
+    author: 'Rauschmayer, Dr. Axel',
+    returnWindow: 'sep, 7, 2020',
+    priceOfTheItem: 31.55,
+    FinalPrice: 34.00,
+    deliveryDate: 'august, 8, 2020',
+    orderPlacedDate: 'august, 4, 2020',
+    shipTo: 'JS Masher',
+    OrderNumber: '114 - 3941689 - 8772232',
+    delivered: true,
+    eligibleForReturn: true,
+    returned: false,
+    productSupport: false,
+    productReview: null,
+    deliveryToResident: true
+  },
+  {
+    item: 'The Timeless Way of Building',
+    author: 'Alexander, Christopher',
+    returnWindow: 'aug, 19, 2020',
+    priceOfTheItem: 41.33,
+    FinalPrice: 44.53,
+    deliveryDate: 'July, 20, 2020',
+    orderPlacedDate: 'July, 19, 2020',
+    shipTo: 'JS Masher',
+    OrderNumber: '113-9984268-1280257',
+    delivered: true,
+    eligibleForReturn: true,
+    returned: false,
+    productSupport: false,
+    productReview: null,
+    deliveryToResident: true
+  },
+  {
+    item: 'Gamecube Controller Adapter. Super Smash Bros Switch Gamecube Adapter for Wii U, PC.Support Turbo and Vibration Features.No Driver and No Lag- Gamecube Adapter',
+    author: null,
+    returnWindow: 'aug, 5, 2020',
+    priceOfTheItem: 15.98,
+    FinalPrice: 17.22,
+    deliveryDate: 'July, 7, 2020',
+    orderPlacedDate: 'July, 4, 2020',
+    shipTo: 'JS Masher',
+    OrderNumber: '114-2875557-9059409',
+    delivered: true,
+    eligibleForReturn: false,
+    returned: false,
+    productSupport: false,
+    productReview: null,
+    deliveryToResident: true
+  },
+  {
+    items: [
+      {
+        item: 'GameCube Controller - Super Smash Bros. Edition (Nintendo Switch)',
+        author: null,
+        returnWindow: 'aug, 4, 2020',
+        priceOfTheItem: 94.95,
+        FinalPrice: 138.93,
+        deliveryDate: 'July, 5, 2020',
+        orderPlacedDate: 'July, 3, 2020',
+        shipTo: 'JS Masher',
+        OrderNumber: '113-2883177-2648248',
+        delivered: true,
+        eligibleForReturn: false,
+        returned: false,
+        productSupport: true,
+        productReview: null,
+        deliveryToResident: false
+      },
+      {
+        item: 'The Art of Sql',
+        author: 'Faroult, Stephane',
+        returnWindow: 'aug, 4, 2020',
+        priceOfTheItem: 33.99,
+        FinalPrice: 138.93,
+        deliveryDate: 'July, 5, 2020',
+        orderPlacedDate: 'July, 3, 2020',
+        shipTo: 'JS Masher',
+        OrderNumber: '113-2883177-2648248',
+        delivered: true,
+        eligibleForReturn: false,
+        returned: false,
+        productSupport: true,
+        deliveryToResident: false
+      }
+    ]
+  }
```

Comment:
The structure of this order does not match the structure of the other orders, which will lead to bugs. You need a consistent format for all the order objects.